instance/broker_test.go

```diff
@@ -119,38 +119,11 @@ var _ = Describe("RunOperation", func() {
 			})
 		})
 
-		Context("when the dashboard URL is not in the correct format", func() {
-			Context("because it is malformed", func() {
-				BeforeEach(func() {
-					fakeCliConnection.GetServiceReturns(plugin_models.GetService_Model{
-						DashboardUrl: "://",
-					}, nil)
-				})
-
-				It("should return a suitable error", func() {
-					Expect(err).To(HaveOccurred())
-					Expect(err).To(MatchError("parse ://: missing protocol scheme"))
-				})
-			})
-
-			Context("because its path format is invalid", func() {
-				BeforeEach(func() {
-					fakeCliConnection.GetServiceReturns(plugin_models.GetService_Model{
-						DashboardUrl: "https://spring-cloud-broker.some.host.name",
-					}, nil)
-				})
-
-				It("should return a suitable error", func() {
-					Expect(err).To(HaveOccurred())
-					Expect(err).To(MatchError("path of https://spring-cloud-broker.some.host.name has no segments"))
-				})
-			})
-		})
-
 		Context("when the dashboard URL is in the correct format", func() {
 			BeforeEach(func() {
 				fakeCliConnection.GetServiceReturns(plugin_models.GetService_Model{
 					DashboardUrl: "https://spring-cloud-broker.some.host.name/x/y/guid",
+					Guid: "guid",
```

Comment:
Was the old version of the test missing this field, or is new for SCS 3? If so we might need a test for when this field doesn't exist?

Reply:
The value should always be there since we get the guid from the model and that model is returned from the cf CLI plugin `code.cloudfoundry.org/cli/plugin/models`.

We were using only the dashboard URL from the returned `GetService_Model` and now, in order to make it compatible with SCS3 we can just use that present field instead of calculating it from the dashboard URL.

So I can't picture a scenario where that value is not present since it comes from an API I didn't consider testing for it.

WDYT?

Reply:
That makes complete sense to me. No point testing for scenarios that don't exist. Approved!